### DISCUSSION TITLE
NativeWind works on App.js, but no other files. But css outputs correctly when using output command

### DISCUSSION THREAD

**Discussion by trevorwrightdev** [nativewind/nativewind#401] [Q&A]
I am just trying to get NativeWind to work as expected, but the css classes only apply on elements in App.js. 

Here is my tailwind config file:
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./App.{js,jsx,ts,tsx}", 
    "./screens/**/*.{js,jsx,ts,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
And here is my App.js file:
```
import { NavigationContainer } from '@react-navigation/native'
import { createNativeStackNavigator } from '@react-navigation/native-stack'
import HomeScreen from './screens/HomeScreen'
import { Text } from 'react-native'

const Stack = createNativeStackNavigator();

export default function App() {
  return (
    <NavigationContainer>
      <Stack.Navigator>
        <Stack.Screen name="Home" component={HomeScreen} />
      </Stack.Navigator>
      <Text className='text-red-500'>testing</Text>
    </NavigationContainer>
  )
}
```

Here is the HomeScreen component:
```
import { View, Text } from 'react-native'

const HomeScreen = () => {
  return (
    <View>
      <Text classname='text-green-500'>helllo whatevr</Text>
    </View>
  )
}

export default HomeScreen

```

The "testing" text tag inside App.js works. It shows up as red in the simulator. But green text class inside the HomeScreen component doesn't work, even though that file is in the screens folder. Also, I know that the Tailwind config file is actually in use because if I remove App.js from the content, tailwind stops working on the testing tag. On top of this, I've done the `npx tailwind -o output.css` command to see if the output css is correct, and it is correct. Both the two text color classes are at the bottom of the file:
```
.text-red-500 {
  --tw-text-opacity: 1;
  color: rgb(239 68 68 / var(--tw-text-opacity));
}

.text-green-500 {
  --tw-text-opacity: 1;
  color: rgb(34 197 94 / var(--tw-text-opacity));
}
```

Why doesn't the green text show up in the simulation? I have also already been doing npx expo start -c to clear any cache. Can anyone help me out?

**Comment by trevorwrightdev** [comment]
hahaha nevermind i got it

**Comment by tkejr** [comment]
How did you get it , I have the same issue ?

**Reply by trevorwrightdev** [reply]
It was just my bad, I was using the word classname instead of className

**Reply by trevorwrightdev** [reply]
As long as your tailwind config file looks like mine you should be good

**Comment by tkejr** [comment]
I actually found the issue I was facing I my tailwind.css.config wasnt configured properly . It worked out haha!

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
